addons/website/models/website.py

```diff
@@ -468,7 +468,14 @@ def configure_page(page_code, snippet_list, pages_views, cta_data):
             page_view_id.save(value=''.join(rendered_snippets), xpath="(//div[hasclass('oe_structure')])[last()]")
 
         def set_images(images):
+            names = self.env['ir.model.data'].search([
+                ('name', '=ilike', 'configurator_%'),
+                ('module', '=', 'website'),
+                ('model', '=', 'ir.attachment')
+            ]).mapped('name')
```

Comment:
Didn't check or test but seems like this will always be true for other websites created on the database if the first one went through the configurator?
Need to at least add the `website_id` part in the domain.

Reply:
@rdeodoo ir_model_data has not related to  website model